codegen/generate_tests.py

```diff
@@ -72,6 +72,13 @@ def test_{entity_snake_case}_roundtrip(instance: {entity_type}) -> None:
     assert instance == result
 """
 
+test_code_java = """\
+@given(instance=from_type({entity_type}))
+@settings(deadline=None)
```

Comment:
suggestion(non-blocking): I've seen some spurious timeouts in Github actions, perhaps we should just disable deadlines globally.

Reply:
Yeah, did this